yarn.lock

```diff
@@ -26440,12 +26440,10 @@ react-dev-utils@^9.0.0:
     strip-ansi "5.2.0"
     text-table "0.2.0"
 
-react-devtools-inline@^4.4.0:
-  version "4.4.0"
-  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.4.0.tgz#e032a6eb17a9977b682306f84b46e683adf4bf68"
-  integrity sha512-ES0GolSrKO8wsKbsEkVeiR/ZAaHQTY4zDh1UW8DImVmm8oaGLl3ijJDvSGe+qDRKPZdPRnDtWWnSvvrgxXdThQ==
-  dependencies:
-    es6-symbol "^3"
+react-devtools-inline@^4.11.1:
+  version "4.11.1"
+  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.11.1.tgz#406cdc370fd416393a81a4654991956d8c7d6157"
+  integrity sha512-pOC3Rscr2S2D1l2NQuDYdC0WHaf1KjW/IB8nBoUtT5AUM1X1yfmbWFANzr1FeT21v4eF9FwjaVpDXnl2kSlRdg==
```

Comment:
```suggestion
react-devtools-inline@4.13.2:
  version "4.13.2"
  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.13.2.tgz#59a5991ab2a946acedd871693a4ceca3ed7f7441"
  integrity sha512-erlHuLtRTClO+B79eu4nyjHGt0vFak1UoDpmSNmA9brF5SsdUTLsmiv6huTnJyNJ2pDKF8nUaekXkRf6E7mwqg==
```

Reply:
Unfortunately, it's still happening the same issue :(
Are you able to run locally and make sure this isn't happening only in the deployment preview?